lib/tasks/events.rake

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+namespace :events do
+  task update_placement_changed_events: :environment do
+    Log.info("Updating placement_changed events...")
+
+    Event
+      .where(type: "placement_changed")
+      .where(changed_field: "status")
+      .where("changed_to::text NOT IN ('\"qualified\"', '\"reserved\"') OR " \
+             "changed_from::text NOT IN ('\"qualified\"', '\"reserved\"')")
+      .where("changed_to::text != '\"disqualified\"' AND changed_from::text != '\"disqualified\"'")
```

Comment:
`changed_to` and `changed_from` have type jsonb

Reply:
won't this be the same as:
```
where("changed_to::text NOT IN ('\"qualified\"', '\"reserved\"', '\"disqualified\"' ) OR " \
             "changed_from::text NOT IN ('\"qualified\"', '\"reserved\"', '\"disqualified\"')"
```
nitpick: what do you think about using heredoc? it should help to remove `\` from the condition